pkg/dt/distributed_transaction_manger.go

```diff
@@ -144,6 +160,7 @@ func (manager *DistributedTransactionManager) BranchRegister(ctx context.Context
 	if err := manager.storageDriver.AddBranchSession(ctx, bs); err != nil {
 		return "", 0, err
 	}
+	metrics.BranchTransactionCounter.WithLabelValues(manager.applicationID, in.ResourceID, metrics.TransactionStatusActive).Inc()
```

Comment:
the status is no correct

Reply:
i think when register a branch session, the branch transaction will be start